concent_api/core/payments/sci_backend.py

```diff
@@ -20,24 +22,24 @@ def get_list_of_payments(
     assert isinstance(provider_eth_address,     str) and len(provider_eth_address)  == ETHEREUM_ADDRESS_LENGTH
     assert isinstance(payment_ts,               int) and payment_ts     >= 0
     assert isinstance(current_time,             int) and current_time   > 0
-    assert isinstance(transaction_type,         str) and transaction_type in ['Force', 'Batch']
+    assert isinstance(transaction_type,         Enum) and transaction_type.name in ['Force', 'Batch']
```

Comment:
This should be `transaction_type in TransactionType`

Reply:
Done